<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitForm</class>
 <widget class="QWidget" name="UnitForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Наименование</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Свойства</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QToolButton" name="btnCreateProperty">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="energo.qrc">
            <normaloff>:/icons/ed519dacc89b2bead3f453b0b05a4a8b.png</normaloff>:/icons/ed519dacc89b2bead3f453b0b05a4a8b.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="btnDeleteProperty">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="energo.qrc">
            <normaloff>:/icons/51efe0e04f50db6a32d32eb749ebb30e.png</normaloff>:/icons/51efe0e04f50db6a32d32eb749ebb30e.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveUnit">
       <property name="text">
        <string>UNDEF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCloseForm">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="energo.qrc"/>
 </resources>
 <connections/>
</ui>
